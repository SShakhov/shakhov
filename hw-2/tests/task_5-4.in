45 39 12 09 67 -6.66 4.2 2.4 -2.4 6.66
