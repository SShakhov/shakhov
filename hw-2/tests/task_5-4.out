172	4.200000
